### DISCUSSION TITLE
error file_get_content

### DISCUSSION THREAD

**Discussion by celvintr** [nk932714/yify-movies-php#40] [General]
Warning: file_get_contents(https://yts.ag/api/v2/list_movies.json?limit=20&page=1&quality=&minimum_rating=&query_term=&genre=&sort_by=&order_by=desc&with_rt_ratings=): Failed to open stream: HTTP request failed! HTTP/1.1 403 Forbidden in C:\laragon\www\yts\index.php on line 146

Fatal error: Uncaught Exception: HTTP request failed. Error was: file_get_contents(https://yts.ag/api/v2/list_movies.json?limit=20&amp;page=1&amp;quality=&amp;minimum_rating=&amp;query_term=&amp;genre=&amp;sort_by=&amp;order_by=desc&amp;with_rt_ratings=): Failed to open stream: HTTP request failed! HTTP/1.1 403 Forbidden in C:\laragon\www\yts\index.php:148 Stack trace: #0 C:\laragon\www\yts\index.php(48): YTS->getFromApi('https://yts.ag/...') #1 C:\laragon\www\yts\index.php(27): YTS->listMovies('', 20, 1, '', '', '', '') #2 {main} thrown in C:\laragon\www\yts\index.php on line 148

**Comment by nk932714** [comment]
try new code
